third_party/move/move-bytecode-verifier/bytecode-verifier-tests/Cargo.toml

```diff
@@ -13,9 +13,9 @@ edition = "2021"
 fail = { workspace = true, features = ['failpoints'] }
 hex = { workspace = true }
 invalid-mutations = { path = "../invalid-mutations" }
```

Comment:
Could be made `{ workspace = true }`?

Reply:
This seems like a testing crate private to prover. Do we want to make it public. I do not know, so left out... Do you think this should better go as a workspace dependency?

Reply:
Feels like an odd one out, @rahxephon89 any reason to keep this private?

Reply:
I am not familiar with this code but I don't think it is for Move prover or at least it is not used by the prover now, more like legacy code?

Reply:
Added to workspace